webview/src/experiments/components/table/CellRowActions.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { MouseEventHandler } from 'react'
```

Comment:
Not sure why, but when I try to hover of the `workspace` radio button, the tooltip appears right on top of it: 

![image](https://user-images.githubusercontent.com/43496356/213748439-b8382bdc-e7c7-47f7-a492-dfabcdd140be.png)